addons/l10n_cl_base/models/res_partner.py

```diff
@@ -0,0 +1,113 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+import logging
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError, ValidationError
+
+_logger = logging.getLogger(__name__)
+
+
+class ResPartner(models.Model):
+    _name = 'res.partner'
+    _inherit = 'res.partner'
+
+    def _get_validation_module(self):
+        self.ensure_one()
+        if not self.country_id:
+            return False, False
+        elif self.l10n_latam_identification_type_id.short_name in ['RUT', 'RUN']:
```

Comment:
Is there a different verification on the RUN than on the RUT?  Otherwise we could just pass both to base_vat